app/controllers/traces_controller.rb

```diff
@@ -54,14 +54,16 @@ def index
 
     @traces = @traces.tagged(params[:tag]) if params[:tag]
 
-    @params = params.permit(:display_name, :tag)
+    @params = params.permit(:display_name, :tag, :min_id, :max_id)
```

Comment:
Most implementations I've seen use "before" and "after" as the parameters. I'm not sure why, but maybe it makes it easier when reasoning about id sort order? (see other comment)

Reply:
I based `max_id` on what we were already doing doing for changesets in the history view but I wasn't especially keen on it.